### PR TITLE
fix: dock panel border color issue

### PR DESCRIPTION
**Pull request by mhduiy**
adjust Inner border and outer border

Issue: https://github.com/linuxdeepin/developer-center/issues/8342

### CODE REVIEW

File: panels/dock/package/main.qml [hunk 1]
```diff
@@ -50,6 +57,8 @@ Window {
     D.DWindow.windowRadius: 0
     D.DWindow.enableBlurWindow: true
     D.DWindow.themeType: Panel.colorTheme
+    D.DWindow.borderWidth: 1
+    D.DWindow.borderColor: D.DTK.themeType === D.ApplicationHelper.DarkType ? Qt.rgba(0, 0, 0, dock.blendColorAlpha(0.4) + 10 / 255) : Qt.rgba(0, 0, 0, 0.15)
```
Comment (18202781743) [inline panels/dock/package/main.qml:61]: 只有暗色需要跟随透明度变化么?

Comment (mhduiy) [reply panels/dock/package/main.qml:61]: 对的, 我把设计最后的图贴上了

**Comment by mhduiy** [comment]
企业微信截图_17187029125889

**Review by 18202781743 — DISMISSED** [review]

**Review by 18202781743 — DISMISSED** [review]

File: panels/dock/package/main.qml [hunk 2]
```diff
@@ -42,6 +42,13 @@ Window {
     color: "transparent"
     flags: Qt.WindowDoesNotAcceptFocus
 
+    function blendColorAlpha(fallback) {
```
[the remark this answers is not in the document]
Comment (mhduiy) [reply panels/dock/package/main.qml:45]: dock里面只有这一个

**Pull request merged**